README.md

```diff
@@ -106,6 +108,27 @@ welcome-page:
     - {{welcome-page}}
 ```
 
+#### Searching components contained in other packages
```

Comment:
I do agree to merge a portion of functionality and address other issues separately but the lib needs to be shippable. In case I'd have to release a patch version or something.

Can we call this experimental for now?
```suggestion
#### [EXPERIMENTAL] Searching components contained in other packages
```